helm/generic-node/values.yaml

```diff
@@ -0,0 +1,115 @@
+---
+name: "generic-node"
+
+testnetName: "testnet"
+
+image:
+  repository: minaprotocol/mina-daemon
+  pullPolicy: IfNotPresent
+  tag: 1.2.1alpha1-1f98b8b-devnet
+
+features:
+  logsnarkworkgossip:
+    enabled: true
+  enablepeerexchange:
+    enabled: true
```

Comment:
false? Ask @nholland94 if peer exchange is required or expected on our mainnet infra